pinot-minion-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/CustomerBasedRetentionTaskExecutor.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.pinot.plugin.minion.tasks;
+
+import org.apache.pinot.core.minion.PinotTaskConfig;
+import org.apache.pinot.minion.exception.TaskCancelledException;
+import org.apache.pinot.minion.executor.PinotTaskExecutor;
+
+public class CustomerBasedRetentionTaskExecutor implements PinotTaskExecutor {
+  protected boolean _cancelled = false;
```

Comment:
doesn't this need to be declared a volatile ?

Reply:
Followed this schema from existing default tasks in latest pinot code.

Reply:
This has to be volatile

Reply:
Not required now so resolving.